tests/unit/wordpress_mock.py

```diff
@@ -517,9 +520,10 @@ def _mock_wp_plugin_deactivate(self, cmd):
 
     @_exec_handler.register(lambda cmd: cmd[:3] == ["wp", "option", "update"])
     def _mock_wp_option_update(self, cmd):
+        # Ignore 1 line summary requirement
```

Comment:
What is the reason for ignoring this? Maybe have a summary line and then put the longer line in the detailed section which can be multiple lines?